README.md

```diff
@@ -861,13 +861,15 @@ registration: {
 ## Registration
 
 > **:warning: Beta feature:** The registration feature is currently a [Beta feature](https://developer.okta.com/docs/api/getting_started/releases-at-okta#beta). This widget functionality won't work unless your Okta organization is part of the Beta program. For help, contact support@okta.com.
+> **:warning: Widget version requirements:** The registration feature requires widget version 2.7.0 or older.
```

Comment:
We should remove this warning. Registration "can" work if the Self Service Registration policy is set.